### ISSUE TITLE
Variables in "Extruder Start/End Gcode" are not substituted for their numeric values

### ISSUE THREAD

**Issue by mpsdskd** [Ultimaker/Cura#2331]
**Tl;dr:** Variables in curly brackets do get substitued in the general Start/End gcodes but not work in the extruder specific gcodes.

I'm trying to get dual extrusion to work on my printer. While the printer changes the material, I want the nozzle to move to the prime tower position, so I put 
_G1 X{prime_tower_position_x} Y{prime_tower_position_y} F2000_
into the "Extruder End Gcode". 
The variable _{prime_tower_position_x}_ in the general "End Gcode" is substituted for its numeric value, but in the extruder specific Gcode the substitution does not work.

I think variables should also be usable in the extruder specific gcodes or are there any reasons not to allow this?

I hope I did not overlook something, but I could not find anything regarding this topic online.

Cura 2.6.1 on Kubuntu Linux 17.04
Custom dual extrusion machine

**Comment by Ghostkeeper** [comment]
It should be substituted: https://github.com/Ultimaker/Cura/commit/341b92860c00ff4cd901a67fe89593f5689906ec

However I see that this is only applied on the global stack, which is rather counterproductive for a setting which only exists on the extruder stacks...
